### PR TITLE
CURA-1919: Adding 3mf file association

### PR DESCRIPTION
**Pull request by thopiekar**
Should work, but need a internal build first to test this, as I can't get own builds working here.

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
Thing is, I can't easily make an internal build of this unless it is merged into master :stuck_out_tongue:

This file association looks identical to its cousins at OBJ and such, so I'd give it a go. What do you think, @awhiemstra? Is something missing? Maybe it explicitly needs a checkbox option somewhere?

File: cmake/NSIS.template.in [hunk 1]
```diff
@@ -555,6 +555,14 @@ Section /o "Open OBJ files with Cura"
     WriteRegStr HKCR "Cura OBJ model file\shell\open\command" "" '"$INSTDIR\Cura.exe" "%1"'
 SectionEnd
 
+Section /o "Open 3MF files with Cura"
+    WriteRegStr HKCR .3mf "" ""
```
Comment (awhiemstra) [inline cmake/NSIS.template.in]: Should add a proper description here, like the other two.

Comment (thopiekar) [inline cmake/NSIS.template.in]: Shouldn't we merge this first and see whether it works?
Adding proper descriptions here should go into a new JIRA issue, I think. CURA-1919 is only about adding 3mf file association.

Comment (awhiemstra) [inline cmake/NSIS.template.in]: If you have a Windows machine, you could simply test it by manually adding the registry keys manually. In any case, I do not think it is a different issue. You add the file association, you should make sure that file association is more descriptive than an empty string.

Comment (sedwards2009) [inline cmake/NSIS.template.in]: "3D Manufacturing Format"

I'll just merge this, add the string and component test it tomorrow morning.

**Pull request merged**